BEP7.md

```diff
@@ -0,0 +1,320 @@
+# BEP-7: Non-fungible Tokens on Binance Chain
+
+- [BEP-7: Non-fungible Tokens on Binance Chain](#bep-7-non-fungible-tokens-on-binance-chain)
+  - [1. Summary](#1-summary)
+  - [2. Abstract](#2-abstract)
+  - [3. Status](#3-status)
+    - [Couple things to keep in mind](couple-things-to-keep-in-mind)
+  - [4. Motivation](#4-motivation)
+  - [5. Collection Specification](#5-collection-specification)
```

Comment:
[]NFT nft token array) is this similar to the index array from ERC-1155 and is collection the semantical version of this concept? 
Non-fungible tokens can be interacted with using an index based accessor into the contract/token data set. Therefore to access a particular token set within a mixed data contract and a particular non-fungible within that set, _id could be passed as <uint128: base token id><uint128: index of non-fungible>.

uint256 baseTokenNFT = 12345 << 128;
uint128 indexNFT = 50;

uint256 baseTokenFT = 54321 << 128;

balanceOf(baseTokenNFT, msg.sender); // Get balance of the base token for non-fungible set 12345 (this MAY be used to get balance of the user for all of this token set if the implementation wishes as a convenience).
balanceOf(baseTokenNFT + indexNFT, msg.sender); // Get balance of the token at index 50 for non-fungible set 12345 (should be 1 if user owns the individual non-fungible token or 0 if they do not).
balanceOf(baseTokenFT, msg.sender); // Get balance of the fungible base token 54321.
https://github.com/ethereum/eips/issues/1155

Reply:
`[]NFT` is golang's implementation, what you are referring to is ethereum's implementation, it is very much different since it's in solidity. The spec here is discussing tech spec, scope only to  
1. what fields
2. what actions can be performed

Reply:
Thanks @amazingandyyy for clarification on this golang vs sol tech spec I wanted to frame a reference point since I am coming from an Etherem Solidity background.

Reply:
yes, I myself was also confused by solc's implementation before.